lib/editing/editor_comid.dart

```diff
@@ -7,33 +7,35 @@ library editor.comid;
 import 'dart:async';
 import 'dart:html' as html;
 import 'dart:math';
+import 'dart:convert';
 
 import 'package:comid/addon/comment/comment.dart' as comments;
 import 'package:comid/addon/edit/closebrackets.dart';
 import 'package:comid/addon/edit/matchbrackets.dart';
-import 'package:comid/addon/edit/show-hint.dart' as hints;
+import 'package:comid/addon/edit/show_hint.dart' as hints;
+import 'package:comid/addon/edit/html_hint.dart' as html;
```

Comment:
perhaps `html_hint`, in order to avoid confusion in the `initialize()` calls below